plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -409,6 +419,28 @@ public void nativeCmd(String name, String pathToExe, List<String> arguments) {
 		LicenseHeaderStep builder;
 		Boolean updateYearWithLatest = null;
 
+		public LicenseHeaderConfig named(String name) {
+			String existingStepName = builder.getName();
+
+			builder = builder.withName(name);
+
+			int existingStepIdx = getExistingStepIdx(existingStepName);
+
+			if (existingStepIdx != -1) {
+				steps.set(existingStepIdx, createStep());
+			} else {
+				addStep(createStep());
+			}
+
```

Comment:
I try not to comment on style issues that Spotless doesn't have rules for, but the blank lines within methods are bothering my eyes a little bit. It's just too different from the style we use elsewhere. Please add a commit that removes blank lines from within the new methods you added.